client/landing/stepper/utils/path.ts

```diff
@@ -45,20 +45,25 @@ export function useLangRouteParam() {
 }
 
 export const useLoginUrl = ( {
-	flowName,
+	variationName,
 	redirectTo,
 	pageTitle,
 	loginPath = `/start/account/user/`,
 }: {
-	flowName?: string;
+	/**
+	 * Variation name is used to track the relevant login flow in the signup framework as explained in https://github.com/Automattic/wp-calypso/issues/67173
+	 */
+	variationName?: string | null;
```

Comment:
Quick question: it stands out here since it's `string | null` instead of `string` like the rest. Could you help me explain the reason?

Reply:
You are right, the others should also accept any falsy value! Fixed in https://github.com/Automattic/wp-calypso/pull/79253/commits/0d2916984a67adb0c9837b06d99077adce8f58d2